src/Home/CreatePlayerButton.js

```diff
@@ -23,9 +24,19 @@ const CreatePlayerButton = (props) => {
           Set Username
         </button>
       </div>
-
       <div>
-        <JoinRoomButton player={player} setPage={setPage}></JoinRoomButton>
+        <CreateRoomButton
+          player={player}
+          setPage={setPage}
+          setRoomCode={setRoomCode}
+        />
+      </div>
+      <div>
+        <JoinRoomButton
+          player={player}
+          setPage={setPage}
+          setRoomCode={setRoomCode}
+        />
       </div>
```

Comment:
`CreateRoomButton` and `JoinRoomButton` are react components, and those components just return JSX. 

You could imagine instead of:
       ```         <CreateRoomButton
          player={player}
          setPage={setPage}
          setRoomCode={setRoomCode}
        />```


this was there instead https://github.com/AnujanAsok/shadow-hunters/blob/9f88889fefaa995c67f06e69cf3b7790dbc2d14e/src/Home/CreateRoomButton.js#L20-L22

which means we probably don't need all the extra divs